src/main-oe/sgxlkl_run_oe.c

```diff
@@ -28,30 +28,33 @@
 #include <netinet/ip.h>
 
 #include "enclave/enclave_mem.h"
+#include "host/serialize_enclave_config.h"
```

Comment:
The sgxlkl_run_oe.c file is becoming very long. It would be good to move some of the functions (e.g. related to config parsing, disk setup etc.) into separate files.

Reply:
I'll leave that to someone else. If we're really rewriting this in C++, this will get a completely new structure anyways.

Reply:
I agree, let's not do this now.